src/Network/Network.Test/Network.Test.csproj

```diff
@@ -14,7 +14,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Graph.RBAC" Version="3.4.0-preview" />
-    <PackageReference Include="Microsoft.Azure.Management.Network" Version="19.8.1-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Network" Version="20.0.0-preview" />
```

Comment:
You need to commit local nuget package (the one in tools\LocalFeed)

Reply:
@number213 Nuget packages have been added.